zerver/openapi/zulip.yaml

```diff
@@ -41,12 +41,13 @@ security:
 paths:
   /fetch_api_key:
     post:
+      operationId: fetch_api_key
+      tags: ["Authentication"]
```

Comment:
I think it makes sense to split this change into two commits, one for tags, and one for `operationId`. It is cleaner.